tree/treeplayer/src/TDFInterface.cxx

```diff
@@ -175,7 +175,7 @@ std::string JitBuildAndBook(const ColumnNames_t &bl, const std::string &prevNode
                             const std::type_info &art, const std::type_info &at, const void *rOnHeap, TTree *tree,
                             const unsigned int nSlots, const std::map<std::string, TmpBranchBasePtr_t> &customColumns)
 {
-   gInterpreter->ProcessLine("#include \"ROOT/TDataFrame.hxx\"");
+   gInterpreter->Calc("#include \"ROOT/TDataFrame.hxx\"");
```

Comment:
This should be `Declare()` as it doesn't need expression evaluation.

Reply:
Fixed